Pod/Classes/PURBufferedOutput.m

```diff
@@ -175,16 +181,9 @@ - (void)callWriteChunk:(PURBufferedOutputChunk *)chunk
                   [[NSNotificationCenter defaultCenter] postNotificationName:PURBufferedOutputDidSuccessWriteChunkNotification object:self];
                   return;
               }
-
-              chunk.retryCount++;
-              if (chunk.retryCount <= self.maxRetryCount) {
-                  int64_t delay = 2.0 * pow(2, chunk.retryCount - 1);
-                  dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(delay * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
-                      [[NSNotificationCenter defaultCenter] postNotificationName:PURBufferedOutputDidRetryWriteChunkNotification object:self];
-
-                      [self callWriteChunk:chunk];
-                  });
-              }
+              
+              // In case of fail, add chunk to the buffer again
+              [self.buffer addObjectsFromArray:chunk.logs];
```

Comment:
As we already discussed offline, let's try to avoid removing functionalities from the framework.